app/controllers/vouchers_controller.rb

```diff
@@ -46,21 +46,32 @@ def new
     @email_disabled = @customer.email.blank?
   end
 
+    #TODO: rediecr to the path where the customer can select showdate herself
+    #find create_voucher from voucher_type, without specfiying showdate_id, skip the reserve_for the showdate, but still
+    #create order, with 
+    #lib/task/staging data
```

Comment:
no commented-out code in PRs please.  if this code needs to be written let's discuss.  if not delete this